cmd/core/main.go

```diff
@@ -116,7 +115,7 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 	setupLog.Info("starting up the core controllers and servers", "git commit", version.String(), "unpacker image", unpackImage)
 
-	dependentRequirement, err := labels.NewRequirement(util.CoreOwnerKindKey, selection.In, []string{rukpakv1alpha1.BundleKind, rukpakv1alpha1.BundleDeploymentKind})
+	dependentRequirement, err := labels.NewRequirement(util.CoreOwnerKindKey, selection.In, []string{rukpakv1alpha2.BundleKind, rukpakv1alpha2.BundleDeploymentKind})
```

Comment:
Since there is no longer a `Bundle` kind, we should probably be using the `BundleDeploymentKind` in this label whenever we create other cluster objects for unpacking (haven't gotten far enough in the review to see if this is the case). But if we're doing that, we no longer need this requirement to include the `BundleKind` label value.
```suggestion
	dependentRequirement, err := labels.NewRequirement(util.CoreOwnerKindKey, selection.In, []string{rukpakv1alpha2.BundleDeploymentKind})
```